<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>3840</width>
    <height>2160</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>UAR - PROJEKT</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2" rowspan="3" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>k</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>300</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Ti</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="spinBoxTi">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spinBoxK">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="spinBoxTd">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Td</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="2" rowspan="3">
     <layout class="QVBoxLayout" name="verticalLayoutPID"/>
    </item>
    <item row="3" column="3" rowspan="3">
     <layout class="QVBoxLayout" name="verticalLayoutUchyb"/>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonARX">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="text">
         <string>ARX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxSposobCalkowania">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Ti poza całką</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ti pod całką</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonResetCalka">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="text">
         <string>Resetuj część całkującą</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QDoubleSpinBox" name="spinBoxAmplituda">
        <property name="minimumSize">
         <size>
          <width>134</width>
          <height>23</height>
         </size>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>134</width>
          <height>123</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>AMPLITUDA</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="minimumSize">
         <size>
          <width>134</width>
          <height>123</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>WYPELNIENIE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="spinBoxWypelnienie">
        <property name="minimumSize">
         <size>
          <width>134</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spinBoxOkres">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>23</height>
         </size>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>123</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>OKRES</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinBoxInterwal">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="signalTypeComboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_14">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>INTERWAŁ</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>ZAPISZ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadButton">
          <property name="text">
           <string>WGRAJ</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>ZADAJ </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="wznowButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>START</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>STOP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>RESET</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnPolaczenie">
        <property name="text">
         <string>Połączenie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRozlacz">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Rozłącz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblStatus">
        <property name="text">
         <string>Tryb lokalny</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
